### ISSUE TITLE
Consider blocklisting of client_characteristic_configuration (00002902-0000-1000-8000-00805f9b34fb)

### ISSUE THREAD

**Issue by Nechoj** [WebBluetoothCG/registries#23]
When trying to connect to a Polar H6 heart-beat measurement device, the [recommended standard procedure](https://googlechrome.github.io/samples/web-bluetooth/notifications.html) failed.

In the [Polar reference implementation for Android](http://developer.polar.com/wiki/H6,_H7,_H10_and_OH1_Heart_rate_sensors#HR_example_code_for_Android)
we can see that the descriptor needs to be modified (search for BluetoothGattDescriptor.ENABLE_NOTIFICATION_VALUE in the example android code).

In summary, WebBluetooth blocks what the manufacturer recommends. This clearly limits the value of WebBluetooth.
